buildstock_query/db_schema/comstock_oedi.toml

```diff
@@ -0,0 +1,32 @@
+[table_suffix]
+baseline = "_metadata"
+timeseries = "_by_state"
+upgrades = "_upgrades"
+
+[column_prefix]
+characteristics = "in."
+output = "out."
+
+[column_names]
+building_id = "bldg_id"
+sample_weight = "weight"
+sqft = "in.sqft"
+timestamp = "timestamp"
+completed_status = "applicability"
+unmet_hours_cooling_hr = ""
+unmet_hours_heating_hr = ""
+fuel_totals = ["out.district_cooling.total.energy_consumption",
```

Comment:
This should exclusively be fuel total columns. Please refer to https://github.com/NREL/buildstock-query/blob/main/buildstock_query/db_schema/resstock_oedi.toml for example.

Reply:
removed district cols